Products/Zuul/facades/__init__.py

```diff
@@ -172,8 +172,6 @@ def getObjectBrains(self, uid=None, start=0, limit=50, sort='name',
         query = None
         globFilters = {}
         prodStates = None
-        if params is None:
-            params = {}
```

Comment:
This should also be restored (or replaced with `params = params if params else {}`.  `getObjectBrains` can be called from other places as well.